W-19783477-rtf-appliance-correlationid-kt (#1574)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/enable-inbound-traffic.adoc

By default, inbound traffic is disabled to prevent consuming unnecessary resources. To allow Mule applications or API gateways to listen on inbound connections, you must enable inbound traffic.

> The CorrelationID for Mule apps running in cluster mode on Runtime Fabric on VMs or bare metal remains a valid UUID. For other deployment targets, the CorrelationID is also prefixed with the cluster node ID.

Reviewer: ```suggestion
The Correlation ID for Mule apps running in cluster mode on Runtime Fabric on VMs or bare metal remains a valid UUID. For other deployment targets, the Correlation ID is also prefixed with the cluster node ID.
```

Or 
"The `CorrelationID` property..."

To avoid passive voice:
"...the Correlation ID includes the cluster node ID as a prefix."

> The CorrelationID for Mule apps running in cluster mode on Runtime Fabric on VMs / Bare metal remains a valid UUID. This differs from other deployment targets where the CorrelationID is also prefixed with the cluster node ID.

Reviewer 2: ```suggestion
The CorrelationID for Mule apps running in cluster mode on Runtime Fabric on VMs or bare metal remains a valid UUID. For other deployment targets, the CorrelationID is also prefixed with the cluster node ID.
```

Reviewer 2: Avoid slashes for translation

== Prepare to Enable Inbound Traffic

> Before enabling inbound traffic for Runtime Fabric on VMs / Bare Metal, be sure you've addressed the following: 

Reviewer 2: avoid slash
